traitsui/testing/tester/wx/helpers.py

```diff
@@ -15,6 +15,62 @@
 from traitsui.testing.tester.exceptions import Disabled
 
 
+def _create_event(event_type, control):
+    """ Creates a wxEvent of a given type 
+
+    Parameters
+    ----------
+    event_type : wxEventType
+        The type of the event to be created 
+    control :
+        The wx control the event is occurring on. 
+
+    Returns
+    -------
+    wxEvent
+        The created event, of the given type, with the given control set as
+        the event object.
+    """
+    event = wx.CommandEvent(
+        event_type, control.GetId()
+    )
+    event.SetEventObject(control)
+    return event
+
+
+def mouse_click(func):
+    """ Decorator function for mouse clicks. Decorated functions will return
+    if they are not enabled. Additionally, this handles the delay for the
+    click.
+
+    Parameters
+    ----------
+    func : callable
+        The mouse click function to be decorated.
```

Comment:
Nit: Although the returned callable has its signature documented, it would be nice to also specify the required signature for the function being decorated: it must accept, at a minimum, the `control` and `delay` as arguments.  `func : callable(*, control, delay, **kwargs)`. This saves the developer from having to parse the code in order to know how to use this decorator.

(Maybe this was also part of what you were asking about earlier... Sorry about that.)